packages/modal/src/types.js

```diff
@@ -1,6 +0,0 @@
-export const types = Object.freeze({
```

Comment:
Why was this removed? These values should be referenced throughout the module.

Reply:
As far as I can tell, it was only being used by storybook previously, not by the library code

Reply:
Added back in 1c5bfed